modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -200,6 +200,8 @@ enum VideoCaptureProperties {
        CAP_PROP_AUDIO_TOTAL_CHANNELS = 64, //!< (read-only) Number of audio channels in the selected audio stream (mono, stereo, etc)
        CAP_PROP_AUDIO_TOTAL_STREAMS = 65, //!< (read-only) Number of audio streams.
        CAP_PROP_AUDIO_SYNCHRONIZE = 66, //!< (open, read) Enables audio synchronization.
+       CAP_PROP_LF_KEY_FRAME=67, //!< Indicates whether the last frame was a key frame or not.
```

Comment:
> last frame

Name doesn't make sense because streams from IP cameras doesn't have the end or duration.

Reply:
Would it be acceptable to rename this to
CAP_PROP_LAST_PACKET_FLAGS
and have it return
packet.flags?

Reply:
I agree, as the flag refers to the packet data stored in the last raw frame would something similar to the below make more sense?
```
CAP_PROP_LAST_RAW_FRAME_CONTAINS_KEY_FRAME
CAP_PROP_LRF_CONTAINS_KEY_FRAME
CAP_PROP_LRF_HAS_KEY_FRAME
CAP_PROP_LRF_HAS_KF
```

Regarding the other property, as the extra data is not necessarily going to be a paramater set, as is the the case of mp4v maybe it makes more sense to call it extra data.
```
CAP_PROP_LRF_PARAM_SET_LEN
CAP_PROP_LRF_EXTRA_DATA_LEN
```

I did try to avoid using extra properties but couldn't think of a way of sending this information inside the raw frame without breaking any code which currently relies on it.